feat: new solution doc updates & contributing updates (#2068)
Azure/Azure-Verified-Modules: docs/content/usage/solution-development/bicep.md

You now have a fully functional Bicep template that will deploy a working Log Analytics workspace! If you would like to try it, run the following in your console:

{{% notice style="note" %}}

> In the below commands you will see that we are using the traditional deployment commands for bicep files to Azure, e.g. `az deployment group create` or `New-AzResourceGroupDeployment`. However, we encourage you to look into converting to using [Deployment Stacks](https://learn.microsoft.com/azure/azure-resource-manager/bicep/deployment-stacks?tabs=azure-powershell) instead by simply replacing the previous commands with `az stack group create` or `New-AzResourceGroupDeploymentStack` as well as the other required input parameters as shown [here](https://learn.microsoft.com/azure/azure-resource-manager/bicep/deployment-stacks?tabs=azure-powershell#create-deployment-stacks).

Reviewer: ```suggestion
For keeping the example below simple, we are using the traditional deployment commands, e.g., `az deployment group create` or `New-AzResourceGroupDeployment`. However, we encourage you to look into using [Deployment Stacks](https://learn.microsoft.com/azure/azure-resource-manager/bicep/deployment-stacks?tabs=azure-powershell) instead by simply replacing the previous commands with `az stack group create` or `New-AzResourceGroupDeploymentStack` as well as the other required input parameters as shown [here](https://learn.microsoft.com/azure/azure-resource-manager/bicep/deployment-stacks?tabs=azure-powershell#create-deployment-stacks).
```
